### PR TITLE
feat: introduce result meta table

### PR DESCRIPTION
**Pull request by lili2311**
Adding in some scanned result meta
Screen Shot 2020-05-05 at 12 06 40

### CODE REVIEW

File: template/test-report.metatable.hbs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<section class="layout-container">
+<table class="metatable">
+    <tbody>
+    <tr class="meta-row"><th class="meta-row-label">Vulnerabilities</th> <td class="meta-row-value">{{uniqueCount}}</td></tr>
+    <tr class="meta-row"><th class="meta-row-label">Dependencies</th> <td class="meta-row-value">{{dependencyCount}}</td></tr>
+    <tr class="meta-row"><th class="meta-row-label">Created on</th> <td class="meta-row-value">{{moment d "MMMM Do YYYY, h:mm:ss a"}}</td></tr>
+    <tr class="meta-row"><th class="meta-row-label">Branch</th> <td class="meta-row-value">{{path}}</td></tr>
```
Comment (gitphill) [inline template/test-report.metatable.hbs:7]: Branch? should this be Path?

File: template/test-report.metatable.hbs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    <tr class="meta-row"><th class="meta-row-label">Tested with</th> <td class="meta-row-value">{{packageManager}}</td></tr>
```
Comment (gitphill) [inline template/test-report.metatable.hbs:8]: Tested with? Should be Package Manager?

File: tap-snapshots/test-snyk-to-html.test.ts-TAP.test.js [hunk 3]
```diff
@@ -325,6 +453,19 @@ exports[`test/snyk-to-html.test.ts TAP template output displays vulns in descend
       </header><!-- .project__header -->
     </div><!-- .layout-stacked__header -->
 
+    <section class="layout-container">
+    <table class="metatable">
+        <tbody>
+        <tr class="meta-row"><th class="meta-row-label">Vulnerabilities</th> <td class="meta-row-value">81</td></tr>
+        <tr class="meta-row"><th class="meta-row-label">Dependencies</th> <td class="meta-row-value">492</td></tr>
+        <tr class="meta-row"><th class="meta-row-label">Created on</th> <td class="meta-row-value">May 5th 2020, 8:52:19 am</td></tr>
```
Comment (gitphill) [inline tap-snapshots/test-snyk-to-html.test.ts-TAP.test.js:461]: are these always the same?

Comment (lili2311) [reply tap-snapshots/test-snyk-to-html.test.ts-TAP.test.js]: removed

File: tap-snapshots/test-snyk-to-html.test.ts-TAP.test.js [hunk 4]
```diff
@@ -308,7 +435,7 @@ exports[`test/snyk-to-html.test.ts TAP template output displays vulns in descend
         <div class="layout-container--short">
             <h1 class="project__header__title">Snyk test summary</h1>
 
-          <p class="timestamp">TIMESTAMP</p>
+          <p class="timestamp">May 5th 2020, 9:43:46 am</p>
```
Comment (gitphill) [inline tap-snapshots/test-snyk-to-html.test.ts-TAP.test.js:438]: leave this as TIMESTAMP

**Pull request merged**